.github/workflows/provider.yaml

```diff
@@ -87,20 +102,14 @@ jobs:
         with:
           timeout_minutes: 3
           max_attempts: 30
-          command: docker-compose -f ops/docker-compose.yaml exec -T lakefs ls lakefs/data/block/data/symlinks/example-repo/example-branch/path/to/symlink.txt
-
-      - name: lakeFS logs
-        if: ${{ always() }}
-        run: docker-compose -f ops/docker-compose.yaml logs --tail=1000 lakefs
+          command: docker exec lakefs ls lakefs/data/block/data/symlinks/example-repo/example-branch/path/to/symlink.txt
```

Comment:
I think this loses the logs?  Or does GitHub keep them for the service container?

Reply:
GitHub dump the services logs at the end as part of the post processing, ex: https://github.com/treeverse/airflow-provider-lakeFS/actions/runs/6533424810/job/17738585733?pr=82